infra/src/main/java/com/redhat/developer/infra/CloudEventExtensions.java

```diff
@@ -0,0 +1,13 @@
+package com.redhat.developer.infra;
+
+public class CloudEventExtensions {
+
+    /**
+     * An extension attribute added to incoming events to identify the target bridge instance
+     */
+    public static final String BRIDGE_ID_EXTENSION = "ebbridgeid";
```

Comment:
Event Bridge Bridge Id...

I was kinda hoping it would easier to namespace the extensions on a CloudEvent, but it seems you can only use a-z or 0-9 according to the spec. Ideally I'd like to have used something like `event-bridge/bridge-id`...but not valid apparently.

I'm happy to change it if you have a better suggestion

Reply:
it's ok, I'm fine with `bridgeid`, `ebridgeid` and `ebbridgeid` :)